ci: update WF version & triggers

vicente-astorga opened the pull request:
- set eol-release on branch conditions
- update WF version to pythonapp-2
- remove paths-ignore conditions

fsologureng commented on .github/workflows/pythonapp.yml:
```diff
@@ -2,23 +2,19 @@ name: Python application
 jobs:
   build:
-    uses: eol-uchile/workflow-commons/.github/workflows/pythonapp.yml@1.0.0
+    uses: eol-uchile/workflow-commons/.github/workflows/pythonapp.yml@2.0.0
```
update to generic major version 2 tag

fsologureng commented on .github/workflows/pythonapp.yml:
```diff
@@ -2,23 +2,19 @@ name: Python application
+      - 'eol-release/*'
 
 jobs:
   build:
```
change job name to test

vicente-astorga committed:
ci: update WF version & triggers
- set eol-release on branch conditions
- update WF version to pythonapp-2
- remove paths-ignore conditions
